app/.env.docker-example

```diff
@@ -26,9 +26,9 @@ DATABASE=postgres
 
 # sqlite settings for local development
 # SQL_ENGINE=
-# SQL_DATABASE=
-# SQL_USER=
-# SQL_PASSWORD=
+# SQL_DATABASE=POSTGRES_DB
```

Comment:
These changes should actually be done in the "postgres settings for docker" section after the other changes rather than here.

The db container needs to read the POSTGRES_DB, etc. variables and our django settings needs to read the SQL_DATABASE, etc. variables. Both sets of variables need to be present for the containers to have the same database credentials and work with one another.

There's no need to change the "sqlite setting..." section.